REUSE.toml

```
@@ -1,6 +1,6 @@
 version = 1
 SPDX-PackageName = "abap-atc-cr-cv-s4hc-tools"
-SPDX-PackageSupplier = "<YOUR-CONTACT (MAIL ADDRESS ETC.)>"
+SPDX-PackageSupplier = "https://github.com/orgs/SAP/teams/abap-atc-cr-cv-s4hc-admin"
```

Hello, SPDX-PackageSupplier should follow the specification at https://spdx.github.io/spdx-spec/v2.3/package-information/#75-package-supplier-field, so it should either be a Person or an Organization.

Examples
* `SPDX-PackageSupplier = "SAP SE (ospo@sap.com)"`
* `SPDX-PackageSupplier = "The SAP Cloud SDK team (cloudsdk@sap.com)"`

```suggestion
SPDX-PackageSupplier = "SAP SE (ospo@sap.com)"
```